scripts.py

```diff
@@ -1374,11 +1374,12 @@ def run_with_arguments(
         if dry_run:
             mirrors = None
 
+        self_hosted_collection = collection_type.is_self_hosted
         replacement_policy = ReplacementPolicy.from_license_source(self._db)
         replacement_policy.mirrors = mirrors
         metadata_records = self.load_metadata(metadata_file, metadata_format, data_source_name, default_medium_type)
         for metadata in metadata_records:
-            self.work_from_metadata(
+            work, licensepool = self.work_from_metadata(
```

Comment:
Since `work` is not used, I think it can be replaced with `_` to not raise any unused variable warnings.